Configuration/NodeTypes.CookieConsent.App.Matomo.yaml

```diff
@@ -0,0 +1,22 @@
+'Sandstorm.CookieConsent:App.Matomo':
+  superTypes:
+    'Sandstorm.CookieConsent:App': true
+  ui:
+    label: Matomo
+  klaro:
+    # This name can be used as a reference in script-tags or fonts that should be lazy-loaded. See klaro documentation
+    name: matomo
+    cookies:
+      - ['_pk_ref', "/"]
+      - ['_pk_cvar', "/"]
+      - ['_pk_id', "/"]
+      - ['_pk_ses', "/"]
+      - ['_pk_hsr', "/"]
+    purposes:
+      - tracking
+  properties:
+    title:
+      defaultValue: Matomo
+    description:
+      defaultValue: Tool zur Analyse von Web-Plattformen.
```

Comment:
Würde mir eine aussagekräftigere Beschreibung wünschen, damit der Nutzer weiß, was Matomo wirklich tut (ich kannte es noch nicht). Auf Matomo.org steht das hier: 

> It provides detailed reports on your website and its visitors, including the search engines and keywords they used, the language they speak, which pages they like, the files they download and so much more. Matomo (Piwik) aims to be an open source alternative to Google Analytics.

Kannste das vll. bisschen in die Beschreibung einfließen lassen?